hydromt/cli/main.py

```diff
@@ -254,20 +260,25 @@ def update(
     quiet,
 ):
     """Update a specific component of a model.
+
     Set an output directory to copy the edited model to a new folder, otherwise maps
     are overwritten.
 
-    \b
     Example usage:
     --------------
 
-    \b
-    Update (overwrite!) landuse-landcover based maps in a Wflow model
-    hydromt update wflow /path/to/model_root -c setup_lulcmaps --opt lulc_fn=vito -d /path/to/data_catalog.yml -v
-
-    \b
-    Update Wflow model components outlined in an .ini configuration file and write the model to a directory
-    hydromt update wflow /path/to/model_root -o /path/to/model_out -i /path/to/wflow_config.ini -d /path/to/data_catalog.yml -v
+    Update (overwrite!) landuse-landcover based maps in a Wflow model:
+    hydromt update wflow /path/to/model_root \
```

Comment:
Same comment as in build, would be nice if the command line example here and below are in one line